Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -214,6 +216,36 @@ final class ProjectGenerator: ProjectGenerating {
         }
     }
 
+    private func generateSwiftPackageReferences(project: Project, pbxproj: PBXProj, pbxProject: PBXProject) throws {
+        var packageReferences: [String: XCRemoteSwiftPackageReference] = [:]
```

Comment:
Similarly to package product, this could also be in `XcodeProj`? I don't think there's going on anything specific to tuist.

Reply:
It's using tuist specific types for `Project` and `Package`. We could look at moving more of this code across if we thought it was a good idea - but again this is much higher level API then what XcodeProj really promises so we would need to carefully consider.

Reply:
OK, I see, let's leave it here then.